packages/@okta/vuepress-site/docs/guides/configure-native-sso/main/index.md

```diff
@@ -0,0 +1,440 @@
+
+> **Note:** All clients that want to leverage Native SSO and SLO must be configured with this grant type.
+
+```json
+  curl --location --request PUT \
+  --url https://${yourOktaDomain}/oauth2/v1/clients/{clientId} \
+  --header 'Content-Type: application/json' \
+  --header 'Accept: application/json' \
+  --header 'Authorization: SSWS <apiKey>' \
+  -d '{
+      "client_id": "{clientId}",
+      "client_id_issued_at": 1618351934,
+      "client_name": "Target Client",
+      "client_uri": null,
+      "logo_uri": null,
+      "redirect_uris": [
+        "https://httpbin.org/get"
+    ],
+    "response_types": [
+       "code"
+    ],
+    "grant_types": [
+        "authorization_code",
+        "refresh_token",
+        "urn:ietf:params:oauth:grant-type:token-exchange"
+    ],
+    "token_endpoint_auth_method": "none",
+    "application_type": "native"
+}'
+```
+
+## Native SSO desktop session lifetime
+
+The device secret assumes the lifetime of the first refresh token that it was minted with. The device secret has the same idle time and maximum time according to the authorization server policy through which it was minted. From there, the device secret and refresh token idle lifetimes are independent of each other.
+
+Other refresh tokens (and other tokens) that are minted by using the device secret are mandated by the authorization server policy through which these tokens are generated. Whenever a device secret is used to generate a new set of tokens, the device secret's idle lifetime or the maximum lifetime is still governed by the original authorization server policy through which the device secret was minted, and it is updated accordingly.
+
+To generate a new set of tokens:
+
+* Use Auth Code with PKCE to obtain the authorization code for the first client.
+* Exchange the code for tokens.
+* Exchange the existing tokens from client 1 for new tokens for client 2.
+
+In this example, you want to SSO to multiple apps that are created by the same company. Each client represents one app, and you can register multiple clients for SSO. When a user signs in to one app, all the other apps that are registered are also automatically signed in.
+
+### Use Authorization Code with PKCE to obtain the authorization code for client 1
+
+Provide the `device_sso`, `openid`, and `offline_access` scopes in the first request to the `/authorize` endpoint using the Authorization Code with PKCE flow. See [Use the Authorization Code flow with PKCE](/docs/guides/implement-auth-code-pkce/use-flow/) for information on the parameters that are being passed in this request.
+
+**Example Authorization Code with PKCE request**
+
+```
+  https://${yourOktaDomain}/oauth2/default/v1/authorize?client_id=0oa10sqibtVca6tme1d7&response_type=code&scope=openid device_sso offline_access&redirect_uri={configured_redirect_uri}&state=state-8600b31f-52d1-4dca-987c-386e3d8967e9&code_challenge_method=S256&code_challenge=qjrzSW9gMiUgpUvqgEPE4_-8swvyCtfOVvg55o5S_es
```

Comment:
I think we should either use placeholders for client id and redirect uri, or we should not use it for both. This mix looks strange. In https://developer.okta.com/docs/guides/implement-auth-code-pkce/use-flow/ we don't use a placeholder value for redirect uri.